src/main/java/com/greghaskins/spectrum/Spectrum.java

```diff
@@ -30,55 +32,73 @@
    * Declare a test suite that describes the expected behavior of the system in a given context.
    *
    * @param context Description of the context for this suite
-   * @param block {@link Block} with one or more calls to {@link #it(String, Block) it} that define
+   * @param block {@link com.greghaskins.spectrum.Block} with one or more calls to {@link #it(String, com.greghaskins.spectrum.Block) it} that define
    *        each expected behavior
    *
    */
-  public static void describe(final String context, final Block block) {
+  public static void describe(final String context, final com.greghaskins.spectrum.Block block) {
     final Suite suite = getCurrentSuiteBeingDeclared().addSuite(context);
-    beginDefintion(suite, block);
+    beginDefinition(suite, block);
+  }
+
+  /**
+   * Declare a test suite that describes the expected behavior of the system in a given context.
+   * The things inside this suite are causally linked, so a failure in one should stop the others
+   * from running.
+   *
+   * @param context Description of the context for this suite
+   * @param block {@link com.greghaskins.spectrum.Block} with one or more calls to
+   *    {@link #it(String, com.greghaskins.spectrum.Block) it} that define each expected behavior
+   *
+   */
+  public static void describeChain(final String context, final com.greghaskins.spectrum.Block block) {
```

Comment:
This is where the requirements of BDD differ to the describe/it syntax. A BDD test should probably stop when a step fails. This allows anyone to say that their Specs within a suite are causally linked.